lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -36,22 +37,15 @@ export async function checkOnboardingBranch(
   setGitAuthor(config.gitAuthor);
   const onboardingPr = await getOnboardingPr(config);
   if (onboardingPr) {
+    logger.debug('Onboarding PR already exists');
     if (config.onboardingRebaseCheckbox) {
       handleOnboardingManualRebase(onboardingPr);
     }
-    logger.debug('Onboarding PR already exists');
-    const { rawConfigHash } = onboardingPr.bodyStruct ?? {};
-    const commit = await rebaseOnboardingBranch(config, rawConfigHash);
-    if (commit) {
-      logger.info(
-        { branch: config.onboardingBranch, commit, onboarding: true },
-        'Branch updated'
-      );
-    }
-    // istanbul ignore if
-    if (platform.refreshPr) {
-      await platform.refreshPr(onboardingPr.number);
-    }
+
+    isConflicted = await scm.isBranchConflicted(
```

Comment:
Should this be part of an `else`? Presumably it won't be conflicted if you just rebased it in the `if`

Reply:
Sorry I don't understand.

Reply:
Since we don't rebase now (explicitly) we need to check it before merging thus I have kept it here.

Reply:
when the if condition above was executed, then we've updated the branch, so isConflicted is always false and we can skip this check.

Reply:
The if-condition above ie. `onboardingRebaseCheckbox` only handles pr-update not branch rebase.

Reply:
oh, ok.